pkg/scheduler/framework/interfaces/types_test.go

```diff
@@ -17,19 +17,17 @@ func TestTargetClustersWithFirstScheduling_Merge(t *testing.T) {
 		result *TargetClusters
 	}{
 		{
-			name: "f1 and f2 in c1 and c2",
+			name: "f1 in c1 and c2",
 			b: &TargetClusters{
 				BindingClusters: []string{"c1", "c2"},
 				Replicas: map[string][]int32{
 					"f1": {1, 1},
-					"f2": {1, 2},
```

Comment:
Can we keep the test cases here unchanged? This will help us identify the new algorithm is stable and do not break previous behaviors.

I think we could add a for-loop for `Feeds` before calling `tcp.MergeOneFeed`.